packages/components/input-number/src/input-number.vue

```diff
@@ -161,54 +165,55 @@ export default defineComponent({
       }
       return precision
     }
-    const _increase = (val: number) => {
+    const handleCompute = (val: number, isIncrease?: boolean) => {
```

Comment:
Instead of passing `isIncrease` can we just pass coefficient to the method? Something like:

```ts
const handleCompute = (val: number, coefficient: 1 | -1 = 1) => {
   if (!isNumber(val)) return data.currentValue
   // Solve the accuracy problem of JS decimal calculation by converting the value to integer.
   val = isNumber(val) ? val : Number.NaN
   return toPrecision(val + props.step * coefficient)
}
```

Reply:
we can rename this method to something like: `withPrecision` or `ensurePrecision`